mobile/src/TableCell.js

```diff
@@ -32,8 +32,8 @@ export default class TableCell extends Component {
               <Avatar user={item.creator} size={20} style={{marginRight: 8, marginLeft: 5}} />
               <Text numberOfLines={2} style={s.nameText}>{item.creator.firstName} {item.creator.lastName}</Text>
               <View style={{flex:1}}></View>
-              <Text style={s.voteText}>{voteCount + " Votes"}</Text>
-              <Text style={s.subText}>{this.props.commentsTotal + " Answer" + ((this.props.commentsTotal === 1) ? "": "s")}</Text>
+              <Text style={s.voteText}>{voteCount + " " + t("votes")}</Text>
```

Comment:
Text containing numbers should be parameterized strings (e.g. `t('votes', {count: voteCount})`)